src/app/Authenticated.tsx

```diff
@@ -76,9 +76,14 @@ export const routeDetails = {
             },
         },
     },
-    registration: {
-        title: 'routeTitle.registration',
-        path: '/register',
+    user: {
+        title: 'routeTitle.user',
+        path: '/user',
+        registration: {
+            title: 'routeTitle.registration',
+            path: 'register',
+            fullPath: '/user/register',
+        },
```

Comment:
Initially this contained password reset but that was removed. I still think we'll need some kind of "user" path so leaving it